src/orion/core/worker/experiment.py

```diff
@@ -781,18 +781,22 @@ def __init__(self, name, user=None):
                              "no view can be created." %
                              (self._experiment.name, self._experiment.metadata['user']))
 
-        try:
-            self._experiment.configure(self._experiment.configuration, enable_branching=False,
-                                       enable_update=False)
-        except ValueError as e:
-            if "Configuration is different and generates a branching event" in str(e):
-                raise RuntimeError(
-                    "Configuration in the database does not correspond to the one generated by "
-                    "Experiment object. This is likely due to a backward incompatible update in "
-                    "Oríon. Please report to https://github.com/epistimio/orion/issues.") from e
-
-            raise
-
+        # TODO: Views are not fully configured until configuration is refactored
+        #       This snippet is to instantiate adapters anyhow, because it is required for
+        #       experiment views in EVC.
+        if self.refers and not isinstance(self.refers.get('adapter'), BaseAdapter):
+            self._experiment.refers['adapter'] = Adapter.build(self.refers['adapter'])
+
+        # try:
```

Comment:
We should remove this commented code. It's on Github, no need to keep it in.

Reply:
I think in this case it is better to keep it there as a strong visual remainder of the work required here once the refactoring of the configuration is done.